src/pke/lib/schemerns/rns-cryptoparameters.cpp

```diff
@@ -122,16 +124,15 @@ void CryptoParametersRNS::PrecomputeCRTTables(KeySwitchTechnique ksTech, Scaling
                 std::make_shared<ILDCRTParams<BigInteger>>(params[0]->GetCyclotomicOrder(), moduli, roots);
         }
 
-        uint32_t sizeP;
         // Find number and size of individual special primes.
         uint32_t maxBits = moduliPartQ[0].GetLengthForBase(2);
-        for (usint j = 1; j < m_numPartQ; j++) {
+        for (uint32_t j = 1; j < m_numPartQ; j++) {
             uint32_t bits = moduliPartQ[j].GetLengthForBase(2);
             if (bits > maxBits)
                 maxBits = bits;
         }
```

Comment:
uint32_t bits = 0, maxBits = 0;
        for (auto&& m : moduliPartQ) {
            if ((bits = m.GetLengthForBase(2)) > maxBits)
                maxBits = bits;
        }